src/main/java/org/medicmobile/webapp/mobile/MedicAndroidJavascript.java

```diff
@@ -262,6 +373,27 @@ public void onDateSet(DatePicker view, int year, int month, int day) {
 		dialog.show();
 	}
 
+	private static HashMap getCPUInfo() throws IOException {
+		BufferedReader bufferedReader = new BufferedReader(new FileReader("/proc/cpuinfo"));
+		String line;
+		HashMap output = new HashMap();
+		while ((line = bufferedReader.readLine()) != null) {
+			String[] data = line.split(":");
+			if (data.length > 1) {
+				String key = data[0].trim();
+				if (key.equals("model name")) {
+					output.put(key, data[1].trim());
```

Comment:
Now that you're no longer counting cores this way you don't need to keep looping if you find the "model name" key. In fact, you don't need to loop at all, and instead use `indexOf`